docs/freqai.md

```diff
@@ -0,0 +1,265 @@
+                for tf in self.freqai_info['timeframes']:
+                        dataframe = self.populate_any_indicators(metadata['pair'],
+                                                                dataframe.copy(), tf)
+                        for i in self.freqai_info['corr_pairlist']:
+                        dataframe = self.populate_any_indicators(i,
+                                        dataframe.copy(), tf, coin=i.split("/")[0]+'-')
+
+                # the model will return 4 values, its prediction, an indication of whether or not the prediction 
+                # should be accepted, the target mean/std values from the labels used during each training period.
+                (dataframe['prediction'], dataframe['do_predict'], 
+                        dataframe['target_mean'], dataframe['target_std']) = self.model.bridge.start(dataframe, metadata)
+
+                return dataframe
+```
+The user should also include `populate_any_indicators()` from `templates/FreqaiExampleStrategy.py` which builds 
+the feature set with a proper naming convention for the IFreqaiModel to use later.
+
+### Building an IFreqaiModel
+
+Freqai has a base example model in `templates/ExamplePredictionModel.py`, but users can customize and create
+their own prediction models using the `IFreqaiModel` class. Users are encouraged to inherit `train()`, `predict()`, 
+and `make_labels()` to let them customize various aspects of their training procedures.
+
+### Running the model live
+
+After the user has designed a desirable featureset, Freqai can be run in dry/live
+using the typical trade command:
+
+```bash
+freqtrade trade --strategy FreqaiExampleStrategy --config config_freqai.example.json --training_timerange '20211220-20220120'
+```
+
+Where the user has now specified exactly which of the models from the sliding window
+that they wish to run live using `--training_timerange` (typically this would be the most
+recent model trained). As of right now, freqai will
+not automatically retain itself, so the user needs to manually retrain and then
+reload the config file with a new `--training_timerange` in order to update the
+model.
+
+
+## Data anylsis techniques
+### Controlling the model learning process
+
+The user can define model settings for the data split `data_split_parameters` and learning parameters
+`model_training_parameters`. Users are encouraged to visit the Catboost documentation
+for more information on how to select these values. `n_estimators` increases the
+computational effort and the fit to the training data. If a user has a GPU
+installed in their system, they may benefit from changing `task_type` to `GPU`.
+The `weight_factor` allows the user to weight more recent data more strongly
+than past data via an exponential function:
+
+$$ W_i = \exp(\frac{-i}{\alpha*n}) $$
+
+where $W_i$ is the weight of data point $i$ in a total set of $n$ data points._
+
+`drop_features` tells Freqai to train the model on the user defined features,
```

Comment:
`drop_features` is not currently in the branch, should be removed from doc or added to branch depending on if the functionality should be retained.